addons/packages/app-toolkit/0.1.0/bundle/config/contour.yaml

```diff
@@ -0,0 +1,34 @@
+#@ load("@ytt:data", "data")
+#@ load("@ytt:yaml", "yaml")
+#@ load("_exclusions.star", "exclusions")
+
+#@ if exclusions.is_pkg_enabled("contour.community.tanzu.vmware.com"):
+
+---
+apiVersion: packaging.carvel.dev/v1alpha1
+kind: PackageInstall
+metadata:
+  name: contour
+  namespace: tanzu-package-repo-global
+  annotations:
+    kapp.k14s.io/change-group: contour
+    kapp.k14s.io/change-rule.serviceaccount: "delete before deleting serviceaccount"
+spec:
+  serviceAccountName: app-toolkit-install-sa
+  packageRef:
+    refName: contour.community.tanzu.vmware.com
+    versionSelection:
+      constraints: 1.19.1
```

Comment:
I would recommend the latest version of Contour, [1.20.1](https://github.com/projectcontour/contour/releases/tag/v1.20.1), as it contains a CVE fix

Reply:
+1 - it shouldn't impact us and only benefits us.